shell/browser/ui/cocoa/NSString+ANSI.mm

```diff
@@ -149,12 +148,11 @@ - (BOOL)containsANSICodes {
 - (NSMutableAttributedString*)attributedStringParsingANSICodes {
   NSMutableAttributedString* result = [[NSMutableAttributedString alloc] init];
 
-  base::scoped_nsobject<NSMutableDictionary> attributes(
-      [[NSMutableDictionary alloc] init]);
+  NSMutableDictionary* __strong attributes([[NSMutableDictionary alloc] init]);
```

Comment:
We don't need a `__strong` here since it's not a member variable (strong is the default, we just add it to member vars for clarity)